src/debugger/mock-ui/debugger-cli.js

```diff
@@ -50,6 +50,12 @@ function readCLIArguments(process, console): DebuggerCLIArguments {
       prepackArguments = args.shift().split(" ");
     } else if (arg === "sourceFile") {
       sourceFile = args.shift();
+    } else if (arg === "diagnosticSeverity") {
+      arg = args.shift();
+      if (arg !== "FatalError" && arg !== "RecoverableError" && arg !== "Warning" && arg !== "Information") {
```

Comment:
You have at least two places doing the validation for severity. You may want to make this as a helper method called "isValidSeverity()", then reuse from both callsites.

Reply:
The checks happen in two distant locations. The best way to share would be to create a utility in `Utils`, and everything currently in there seems more involved than a two line check. I think creating/importing such a utility would create more confusion than clarity.